packages/server-side-graphql-client/tests/index.test.js

```diff
@@ -0,0 +1,225 @@
+const { Text, Integer } = require('@keystonejs/fields');
+const { multiAdapterRunners, setupServer } = require('@keystonejs/test-utils');
+
+const {
+  createItems,
+  createItem,
+  deleteItem,
+  deleteItems,
+  getItemById,
+  getItems,
+  updateItem,
+  updateItems,
+} = require('../lib/server-side-graphql-client');
+
+const testData = [
+  {
+    data: {
+      name: 'test',
+      age: 30,
+    },
+  },
+  { data: { name: 'test2', age: 40 } },
+];
+const schemaName = 'testing';
+
+const seedDb = ({ keystone, ...rest }) =>
+  createItems({
+    keystone,
+    listName: 'Test',
+    items: testData,
+    schemaName,
+    ...rest,
+  });
+
+function setupKeystone(adapterName) {
+  return setupServer({
+    adapterName,
+    createLists: keystone => {
+      keystone.createList('Test', {
+        fields: {
+          name: { type: Text },
+          age: { type: Integer },
+        },
+      });
+    },
+  });
+}
+
+multiAdapterRunners().map(({ runner, adapterName }) =>
+  describe(`Adapter: ${adapterName}`, () => {
+    describe('create', () => {
+      test(
+        'Should create and get single item',
+        runner(setupKeystone, async ({ keystone }) => {
+          // Seed the db
+          const item = await createItem({
+            keystone,
+            listName: 'Test',
+            item: testData[0].data,
+            schemaName,
+          });
+          expect(typeof item.id).toBe('string');
+
+          // Get single item from db
+          const getItem = await getItemById({
+            keystone,
+            listName: 'Test',
+            returnFields: 'name, age',
+            itemId: item.id,
+            schemaName,
+          });
+
+          expect(getItem).toEqual(testData[0].data);
+        })
+      );
+      test(
+        'Should create and get multiple items',
+        runner(setupKeystone, async ({ keystone }) => {
+          // Seed the db
+          await seedDb({ keystone });
+
+          // Get all the items back from db
+          const allItems = await getItems({
+            keystone,
+            listName: 'Test',
+            returnFields: 'name, age',
+            schemaName,
+          });
+
+          expect(allItems).toEqual(testData.map(x => x.data));
+        })
+      );
+    });
+    describe('udpate', () => {
+      test(
+        'Should update single item',
+        runner(setupKeystone, async ({ keystone }) => {
+          // Seed the db with single item
+          const seedItems = await seedDb({ keystone, item: testData[0].data });
```

Comment:
```suggestion
          const seedItems = await seedDb({ keystone });
```

Reply:
I guess this test works ok without that, I'd assumed Aman had done this for a reason - but if not we can remove that `...rest`

Reply:
Well this code doesn't actually do anything. Passing `item` into `createItems` doesn't ever get used 🤷‍♂️

Reply:
Correct, should be `items` but I still think we can just remove -I'll do it and see if CI blows up :)